### PR TITLE
added images and readme changes

### PR DESCRIPTION
**Pull request by skulshrestha-pdftron**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,112 @@
...
+- [Project Components](#project-components)
+- [Overall Flow of Information](#overall-information-flow)
+<!-- - Contributions section perhaps? -->
+
+## Technologies/Concepts
+
+### Technologies
+
+- Raspberry Pi
+- <a href="https://github.com/ageitgey/face_recognition" target="_blank"><code>face_recognition</code> Python library</a>
```
Comment (lcfyi) [inline README.md]: It might be better to use markdown link formatting instead.

**Review by lcfyi — COMMENTED** [review]
I only had a quick skim of the document, but it looks pretty good so far. 

GitHub allows for some HTML formatting that's "safe", but I prefer purely markdown so it displays properly even when you pull it.

File: README.md (hunk 1 above)
Comment (ShreyansK2000) [reply README.md]: It's just easier to do the new tab opening stuff. I can fix it still if you prefer it that way

Comment (lcfyi) [reply README.md]: You mean to be able to click on images? 

You can chain markdown:

[alt_text](https://i.imgur.com/GGTkAvq.gif)


`[alt_text](whatever link)`

**Pull request merged**